test/path.jl

```diff
@@ -106,22 +116,24 @@ mktmpdir() do d
                 @test_throws ErrorException chown(p"newfile", "nobody", "nogroup"; recursive=true)
             end
 
-            chmod(p"newfile", user=(READ+WRITE+EXEC), group=(READ+EXEC), other=READ)
-            @test String(mode(p"newfile")) == "-rwxr-xr--"
-            @test isexecutable(p"newfile")
-            @test iswritable(p"newfile")
-            @test isreadable(p"newfile")
-
-            chmod(p"newfile", "-x")
-            @test !isexecutable(p"newfile")
-
-            @test String(mode(p"newfile")) == "-rw-r--r--"
-            chmod(p"newfile", "+x")
-            write(p"newfile", "foobar")
-            @test read(p"newfile") == "foobar"
-            chmod(p"newfile", "u=rwx")
-
-            chmod(new_path, mode(p"newfile"); recursive=true)
+            @static if is_unix()
```

Comment:
It'd be nice to at least partially support `chmod` on windows similar to python, but that's probably better as a separate change.

Reply:
Yeah, we can fix that later: https://github.com/rofinn/FilePaths.jl/issues/20